src/Overlay.react.js

```diff
@@ -95,7 +99,11 @@ const Overlay = React.createClass({
 
   _updatePosition() {
     // Positioning is only used when body is the container.
-    if (!isBody(this.props.container)) {
+    if (
+      !this.props.show ||
+      !this.isMounted() ||
```

Comment:
Isn't use of `isMounted` discouraged? https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html